src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private static final FigureName figureName = FigureName.SQUARE;
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure:\t" + figureName
```

Comment:
Don't use `\n` at the beginning of the printed line. It's better to control line breaks from the outside of the `draw()` method if necessary.